### ISSUE TITLE
Simulate Database delay

### ISSUE THREAD

**Issue by bhansconnect** [the-benchmarker/web-frameworks#2181]
Most of the time, the hard task of a web server is to be able to keep track of many concurrent connections that are all waiting on the database. With that kind of load, it is easy for the web server to reach a point where it can no longer handle the amount of connections and everything dramatically slows.

It would be nice to have a test where the web servers sleep for 0.5 seconds and then returns a simple or empty packet. This will show how well the web servers deal with the async nature of the web(database, crypto for login, other apis). 

Side note, I would personally I would advise using [wrk2](https://github.com/giltene/wrk2)

**Comment by bhansconnect** [comment]
Instead of actually testing the database, I would advise just adding a sleep into one of the endpoints in order to simulate needing to wait on a database or some sort of external api. That would be simple and will show which frameworks better support asynchronous execution. Given how often web servers have to wait on an external dependency, it will show more accurate performance results with some form of simulated delay.

**Comment by waghanza** [comment]
could you please expose your point in https://github.com/the-benchmarker/web-frameworks/issues/1728 @bhansconnect 

seems not to be a bad idea, thought
